### PR TITLE
dj-rest-auth based password reset

### PR DESCRIPTION
**Pull request by anpandey**

### CODE REVIEW

File: src/planscape/planscape/settings.py [hunk 1]
```diff
@@ -108,6 +108,9 @@
 
 WSGI_APPLICATION = "planscape.wsgi.application"
 
+# TODO: This is for debugging email messages, and it needs to be removed.
```
Comment (janicejl) [inline src/planscape/planscape/settings.py:111]: Can be removed after pull from main.

File: src/planscape/planscape/urls.py [hunk 2]
```diff
@@ -29,4 +30,8 @@
     path('planscape-backend/dj-rest-auth/', include('dj_rest_auth.urls')),
     path('planscape-backend/dj-rest-auth/registration/',
          include('dj_rest_auth.registration.urls')),
+    # TODO: We don't actually need this since we'll be using
+    # a different endpoint.
+    path('planscape-backend/dj-rest-auth/password/reset/confirm/<uidb64>/<token>/',
```
Comment (janicejl) [inline src/planscape/planscape/urls.py:35]: Is this uint64?

Comment (sterlingwellscaffeine) [reply src/planscape/planscape/urls.py:35]: I'm confused about the TODO that is here. Why is this being checked in if we are not using this endpoint?

**Review by sterlingwellscaffeine — CHANGES_REQUESTED** [review]

File: src/planscape/planscape/settings.py (hunk 1 above)
Comment (bwndai) [reply src/planscape/planscape/settings.py:111]: +1

File: src/planscape/planscape/urls.py (hunk 2 above)
Comment (anpandey) [reply src/planscape/planscape/urls.py]: This was a temporary solution that doesn't apply anymore. I've removed the change.

**Review by sterlingwellscaffeine — APPROVED** [review]

**Pull request merged**